.github/workflows/main.yml

```diff
@@ -84,7 +83,20 @@ jobs:
           rm -rf texar-pytorch
       - name: Install Forte
         run: |
-          pip install --use-feature=in-tree-build --progress-bar off .[ner,test,example,wikipedia,augment,stave,audio_ext,remote]
+          pip install --use-feature=in-tree-build --progress-bar off .[models,test,wikipedia,data_aug,nlp,ir,texar-encoder,stave,audio_ext,remote,extractor]
+      - name: Test backbone Forte import test
+        run: |
+          # Try to install Forte backbone only and test basic imports.
+          pip install --use-feature=in-tree-build --progress-bar off .
+          # needs to remove it in after torch dependency is removed
+          pip uninstall -y texar-pytorch
+          pytest tests/forte/texar_nondependency_test.py
+          # install lastest texar pytorch
+          git clone https://github.com/asyml/texar-pytorch.git
+          cd texar-pytorch
+          pip install --progress-bar off .
+          cd ..
+          rm -rf texar-pytorch
```

Comment:
why don't we  conduct these steps before the `Install Texar` or even the `Install deep learning frameworks` step?

Reply:
Since we still have dependencies on other libraries in the dependency matrix. In this PR, we only need to remove `texar`. In the later PR that we set up a dependency env for different modules, these lines of code will be removed.